.github/workflows/patch-images.yaml

```diff
@@ -0,0 +1,69 @@
+on:
+  schedule:
+    - cron: "0 0 * * *" # nightly
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  patch:
+      runs-on: ubuntu-latest
+      strategy:
+        fail-fast: false
+        matrix:
+          images: ['ghcr.io/azure/azure-workload-identity/proxy-init:latest-linux-arm64', 'ghcr.io/azure/azure-workload-identity/proxy-init:latest-linux-amd64']
+      steps:
```

Comment:
Could you add the `Harden runner` step

https://github.com/Azure/azure-workload-identity/blob/eb7cef39ec3bae6a49f02c90a3fbb257b156f7c6/.github/workflows/publish-images.yaml#L38-L41